interface/VideoDecoderDefs.h

```diff
@@ -57,6 +57,13 @@ typedef struct {
     VAProfile profile;
     uint32_t flag;
     uint32_t fourcc;
+    //xxxLayer - layer number, 0: decode all layers, >0: decode up to target layer.
```

Comment:
Please remove unsupport things. vp8 not support it, not mean all codecs not support it.